setup.py

```diff
@@ -16,7 +16,7 @@
     author_email='info@polyswarm.io',
     url='https://github.com/polyswarm/polyswarm-artifact',
     license='MIT',
-    python_requires='>=3.5.2,<4',
+    python_requires='>=2.7,<4',
```

Comment:
also, by saying >=2.7 and <4, we are including 3.0-3.4. Do you know of any problems with 3.0-3.4 such that we do NOT want to support them?

Reply:
I'm not aware of any such problems, that was intentional. Something funky with `enum`s in 2.7 though, still figuring it out.

Reply:
Looks like they were added in 3.4